decidim-elections/app/cells/decidim/votings/content_blocks/metrics_cell.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Votings
+    module ContentBlocks
+      # VotingsMetricChartsPresenter is not implemented yet. This cell won't
```

Comment:
There's a linter error here: 
```suggestion
      # VotingsMetricChartsPresenter is not implemented yet. This cell will not
```